ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2574,6 +2574,35 @@ def test_jax_numpy_around(
     )
 
 
+# round
+@handle_frontend_test(
+    fn_tree="jax.numpy.round",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["float64"],
```

Comment:
why the available_dtype here is only one, you should use the helpers.get_dtypes() function instead to cover all cases.